### ISSUE TITLE
Submission on server

### ISSUE THREAD

**Issue by John-Yao** [ifzhang/FairMOT#301]
Hi, I try to submit the baseline result into server.

Following testing instructions, I got the below errors after submitting the result to server.

Wrong number of result files! 7 results found in archive, but 42 needed. Please make sure sure that all required txt files are in the root folder of the zip archive.

Could you provide some suggestion.

**Comment by ifzhang** [comment]
You can run 7 training set results and 7 test set results and then copy them for the other 28 files and rename them. For example, MOT17-01-SDP.txt, MOT17-01-DPM.txt, MOT17-01-FRCNN.txt have the same results.

**Comment by John-Yao** [comment]
Thanks for you reply!
